mkdocs.yml

```diff
@@ -474,6 +475,7 @@ nav:
               - Software Image File: "user-guide/core-data-model/dcim/softwareimagefile.md"
               - Software Version: "user-guide/core-data-model/dcim/softwareversion.md"
               - Virtual Chassis: "user-guide/core-data-model/dcim/virtualchassis.md"
+              - Interface VDC Assignment: "user-guide/core-data-model/dcim/interfacevdcassignment.md"
```

Comment:
I'd suggest making the documentation for this model a part of `virtualdevicecontext.md` rather than a separate file, personally.